src/unix/fs.c

```diff
@@ -858,9 +858,14 @@ static ssize_t uv__fs_buf_iter(uv_fs_t* req, uv__fs_buf_iter_processor process)
     total += result;
   }
 
+  if (errno == EINTR && total == -1)
+    return total;
```

Comment:
You can't get to this line with `total == -1` after the first iteration, can you? Since 849 only sets `total` to `result` if `total == 0`.